config/drupal-9/drupal-9.0-deprecations.php

```diff
@@ -15,14 +15,20 @@
         return new AddCommentService();
     });
 
+    if (defined('\Rector\Symfony\Set\TwigSetList::TWIG_24')) {
+        $twigSet = TwigSetList::TWIG_24;
+    } else {
+        $twigSet = TwigSetList::TWIG_240;
+    }
```

Comment:
Also `use Rector\Exception\ShouldNotHappenException;` should be added.

```suggestion
    if (defined(TwigSetList::class .'::TWIG_24')) {
        $twigSet = TwigSetList::TWIG_24;
    } elseif (defined(TwigSetList::class .'::TWIG_240'))  {
        $twigSet = TwigSetList::TWIG_240;
    } else {
        throw new ShouldNotHappenException('Could not detect twig set.');
    }
```